### PR TITLE
Specializes DigitalSurface::facesAroundVertex in the 3D case

### PR DESCRIPTION
**Pull request by JacquesOlivierLachaud**
# PR Description

This PR specializes the method DigitalSurface::facesAroundVertex in the 3D case, such that faces (ie pointels) are ordered counterclockwise with respect of the vertex (ie surfel) seen from the exterior.

# Checklist

- [x] Unit-test of your feature by updating topology/testDigitalSurface
- [x] Doxygen documentation of the code completed (classes, methods, types, members...)
- [x] Documentation module page added or updated.
- [x] New entry in the [ChangeLog.md](https://github.com/DGtal-team/DGtal/blob/master/ChangeLog.md) added.
- [x] No warning raised in Debug ```cmake``` mode (otherwise, Travis C.I. will fail).
- [x] All continuous integration tests pass (Travis & appveyor)

### CODE REVIEW

**Comment by dcoeurjo** [comment]
ping @JacquesOlivierLachaud , I'd like to merge this PR ;)

File: ChangeLog.md [hunk 1]
```diff
@@ -85,6 +85,8 @@
   - Small fixes in Shortcuts and ShortcutsGeometry, doc, and colormaps.
     (Jacques-Olivier Lachaud, [#1364](https://github.com/DGtal-team/DGtal/pull/1364))
 
+- *Topology*
+  - Specializes the method DigitalSurface::facesAroundVertex in the 3D case, such that faces (ie pointels) are ordered counterclockwise with respect of the vertex (ie surfel) seen from the exterior. (Jacques-Olivier Lachaud, [#1377](https://github.com/DGtal-team/DGtal/pull/1377))
```
[the remark this answers is not in the document]
Comment (JacquesOlivierLachaud) [reply ChangeLog.md]: A surfel is oriented and its direct incidence tells where is the interior. Do you really want that I precise this concept here in the ChangeLog.

[the remark this answers is not in the document]
Comment (dcoeurjo) [reply ChangeLog.md]: and if the surface has boundaries / holes ?

[the remark this answers is not in the document]
Comment (dcoeurjo) [reply ChangeLog.md]: constructed explicitly.?

[the remark this answers is not in the document]
Comment (JacquesOlivierLachaud) [reply ChangeLog.md]: Then in this case, this consistency rule is valid only locally

File: src/DGtal/topology/DigitalSurface.h [hunk 2]
```diff
@@ -467,11 +468,18 @@ namespace DGtal
 
     /**
        @param v any vertex (surfel) of the surface.
+
+       @param order_ccw_in_3d when 'true', orders faces
+       counterclockwise around vertex (solely in 3d). It corresponds
+       to ordering the pointels counterclockwise around the given
```
[the remark this answers is not in the document]
Comment (JacquesOlivierLachaud) [reply src/DGtal/topology/DigitalSurface.h]: I have updated the doc here and in class DigitalSurface

[the remark this answers is not in the document]
Comment (dcoeurjo) [reply src/DGtal/topology/DigitalSurface.h]: Perfect. Many thanks

**Comment by dcoeurjo** [comment]
All good. Let me see if I can fix the conflict for you...

**Comment by dcoeurjo** [comment]
Conflict ok but I don’t know why the tests are not launched...

**Pull request merged**